samples/OcelotServiceDiscovery/DownstreamService/Program.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.AspNetCore;
+using Microsoft.AspNetCore.Hosting;
+using System;
+
+namespace Ocelot.Samples.ServiceDiscovery.DownstreamService;
+
+public class Program
+{
+    public static void Main(string[] args)
+    {
+        BuildWebHost(args).Run();
+    }
+
+    public static IWebHost BuildWebHost(string[] args) =>
+        WebHost.CreateDefaultBuilder(args)
+            .UseUrls($"http://{Environment.MachineName}:5001")
```

Comment:
Pretty hardcoded!
Can we remote it and use appropriate setting only in **launchSettings.json** file?